### PR TITLE
Mac 93 move endpoints from accounts to finance

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.EmployerFinance/Queries/GetEmployerAccountTransactions/GetEmployerAccountTransactionsHandler.cs [hunk 1]
```diff
@@ -45,12 +45,7 @@ public async Task<GetEmployerAccountTransactionsResponse> Handle(GetEmployerAcco
             if (!result.IsValid())
             {
                 throw new InvalidRequestException(result.ValidationDictionary);
-            }
-
-            if (result.IsUnauthorized)
-            {
-                throw new UnauthorizedAccessException();
-            }
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerFinance/Queries/GetEmployerAccountTransactions/GetEmployerAccountTransactionsHandler.cs:53]: Should this have been removed?

File: src/SFA.DAS.EmployerFinance/Queries/GetEmployerAccountTransactions/GetEmployerAccountTransactionsValidator.cs [hunk 2]
```diff
@@ -28,12 +18,7 @@ public async Task<ValidationResult> ValidateAsync(GetEmployerAccountTransactions
             {
                 result.AddError(nameof(item.HashedAccountId), "HashedAccountId has not been supplied");
             }
-
-            if (result.IsValid() && !string.IsNullOrEmpty(item.ExternalUserId))
-            {
-                result.IsUnauthorized = !_authorizationService.IsAuthorized(EmployerUserRole.Any);
-            }
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerFinance/Queries/GetEmployerAccountTransactions/GetEmployerAccountTransactionsValidator.cs:35]: should we be removing the authorization?

**Review by cofaulco — COMMENTED** [review]
couple of questions

**Review by cofaulco — APPROVED** [review]

File: src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj [hunk 3]
```diff
@@ -100,6 +101,10 @@
       <Project>{05D7C094-E2C9-4CCD-B0D7-85D263254B26}</Project>
       <Name>SFA.DAS.EAS.TestCommon</Name>
     </ProjectReference>
+    <ProjectReference Include="..\SFA.DAS.EmployerFinance\SFA.DAS.EmployerFinance.csproj">
+      <Project>{C5880F73-59D6-4C59-A1A2-DBA38D13E8DC}</Project>
+      <Name>SFA.DAS.EmployerFinance</Name>
+    </ProjectReference>
```
Comment (cofaulco) [inline src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj:107]: EAS Api shouldn't have a direct reference to FInance.csproj else we won't be able to split to separate codebases.

**Review by cofaulco — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj (hunk 3 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj]: remove the ref

**Review by cofaulco — APPROVED** [review]

**Pull request merged**